source/utilities.tex

```diff
@@ -9372,7 +9372,7 @@
 
 \begin{itemdescr}
 \pnum
-\expects
+\hardexpects
```

Comment:
Missing 2 of these edits around line 9418 for [expected.void.obs]/p7 and line 9435 for expected.void.obs/p9.

Reply:
Thanks, fixed.